esp/src/eclwatch/WUDetailsWidget.js

```diff
@@ -156,16 +156,23 @@ define([
 
             _onCancelDialog: function () {
                 this.zapDialog.hide();
-                this.emailCheckbox.set("checked", false);
+                this.includeSlaveLogsCheckbox.set("value", "off");
+                this.includeSlaveLogsCheckbox.set("checked", false);
                 this.emailCheckbox.set("value", "false");
                 this.emailSubject.reset();
                 this.emailBody.reset();
             },
 
             _onSubmitDialog: function(){
+                var context = this;
+                var zapForm = this.zapForm.value;
+                lang.mixin(zapForm, {
+                    IncludeThorSlaveLog: ([this.includeSlaveLogsCheckbox.set("checked",true), this.includeSlaveLogsCheckbox.value]),
```

Comment:
This does not look correct?  `IncludeThorSlaveLog` should be either "off" or "on" (and if omitted it defaults to "on").

IOW it should simply be:  `IncludeThorSlaveLog: this.includeSlaveLogsCheckbox.get("value")`
Or 
It could have been (without all the messing with "value":  
```
IncludeThorSlaveLog:  this.includeSlaveLogsCheckbox.get("checked") ? "on" : "off"`
```

Reply:
changes made after discussing with @GordonSmith